pkg/serviceaccounts/controllers/docker_registry_service.go

```diff
@@ -248,6 +251,7 @@ func (e *DockerRegistryServiceController) getDockerRegistryLocations() []string
 		return []string{
 			net.JoinHostPort(service.Spec.ClusterIP, fmt.Sprintf("%d", service.Spec.Ports[0].Port)),
 			net.JoinHostPort(fmt.Sprintf("%s.%s.svc", service.Name, service.Namespace), fmt.Sprintf("%d", service.Spec.Ports[0].Port)),
+			net.JoinHostPort(fmt.Sprintf("%s.%s.svc.%s", service.Name, service.Namespace, defaultClusterDNSName), fmt.Sprintf("%d", service.Spec.Ports[0].Port)),
```

Comment:
@deads2k how you can configure it? I mean even if you configure it for origin, will it work? (it is hardcoded in pkg/dns for example.